tests/resources/test_transformation_request.py

```diff
@@ -84,26 +84,32 @@ def test_submit_transformation_bad_result_format(self, mocker, mock_rabbit_adapt
         response = client.post('/servicex/transformation', json=request)
         assert response.status_code == 400
 
-    def test_submit_transformation_bad_workflow(self, mocker, mock_rabbit_adaptor):
-        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor)
+    def test_submit_transformation_bad_workflow(self, mock_rabbit_adaptor,
+                                                mock_docker_repo_adapter):
+        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor,
+                                   docker_repo_adapter=mock_docker_repo_adapter)
         request = self._generate_transformation_request()
         request['columns'] = None
         request['selection'] = None
 
         r = client.post('/servicex/transformation', json=request)
         assert r.status_code == 400
 
-    def test_submit_transformation_request_throws_exception(self, mocker, mock_rabbit_adaptor):
+    def test_submit_transformation_request_throws_exception(self, mocker,
+                                                            mock_rabbit_adaptor,
+                                                            mock_docker_repo_adapter):
         mock_rabbit_adaptor.setup_queue = mocker.Mock(side_effect=Exception('Test'))
-        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor)
+        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor,
+                                   docker_repo_adapter=mock_docker_repo_adapter)
 
         response = client.post('/servicex/transformation',
                                json=self._generate_transformation_request())
         assert response.status_code == 500
         assert response.json == {"message": "Something went wrong"}
```

Comment:
I would like a issue whose goal is to eliminate all "something went wrong" messages back to the user. :-)

Reply:
Added to ServiceX Issue 105